Dockerfile

```diff
@@ -0,0 +1,66 @@
+# NAME: shecancode/hatch-site
+
+# Use the barebones version of Ruby 2.4.1.
+FROM ruby:2.4.1-slim
+
+
+# Setting environment variables using sane defaults
+# These can be overwritten at run time
+
+ENV RAILS_DB_HOST localhost
+ENV RAILS_DB_PORT 5432
+ENV RAILS_DB_USER postgres
+ENV RAILS_DB_PASS changeme
+
+ENV RAILS_ADMIN_USER_EMAIL admin@example.com
+ENV RAILS_ADMIN_USER_FIRSTNAME Admin
+ENV RAILS_ADMIN_USER_LASTNAME User
+ENV RAILS_ADMIN_USER_PASS changeme
+
+# Install dependencies:
+# We need to add yarn to apt sources
+# + use the correct version of node
+
+ADD https://dl.yarnpkg.com/debian/pubkey.gpg /tmp/yarn-pubkey.gpg
+RUN apt-key add /tmp/yarn-pubkey.gpg && rm /tmp/yarn-pubkey.gpg
+RUN echo 'deb http://dl.yarnpkg.com/debian/ stable main' > /etc/apt/sources.list.d/yarn.list
+RUN apt-get update && apt-get install -qq -y --no-install-recommends \
+      build-essential libpq-dev curl
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get update && apt-get install -qq -y --no-install-recommends nodejs yarn
+
+# This sets the context of where commands will be ran
+RUN mkdir -p /hatch_web
+WORKDIR /hatch_web
+
+# Use COPY for copying local files
+# relative paths are:
+# first arg: relative to the path passed to `docker build`
+# second arg: relative to WORKDIR
+COPY Gemfile Gemfile
```

Comment:
Proposal (not mandatory at all) of improvement for this line:
`COPY ./Gemfile /hatch_web/Gemfile`

=> It will avoid ambiguity, and make explicit some part of the knowledge of the comment

Don't worry about the maintenability (when changing the value /hatch_web), we can make this as a variable later